CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - [sdk/dotnet] Fix creation of CustomResources (https://github.com/pulumi/pulumi-kubernetes/pull/1741)
 - Always override namespace for helm release operations (https://github.com/pulumi/pulumi-kubernetes/pull/1747)
 - Add k8s client tuning settings to Provider (https://github.com/pulumi/pulumi-kubernetes/pull/1748)
+- Nest helm.Release Provider settings (https://github.com/pulumi/pulumi-kubernetes/pull/1751)
```

Comment:
Could you elaborate with a couple of examples for setting nested config entries? I don't think the `pulumi config set --path` option is well advertised.